packages/core/src/feature-app-manager.ts

```diff
@@ -45,8 +45,11 @@ export interface FeatureAppEnvironment<
    * If this callback is defined, a short-lived Feature App can call this
    * function when it has completed its task. The Integrator (or parent Feature
    * App) can then decide to e.g. unmount the Feature App.
+   *
+   * By default it is assumed that the FA exited successfully, but it can signal
+   * that it failed. It is up to the integrator to handle this.
```

Comment:
We should update this text now. Suggestion:

> Optionally, the Feature App can pass a `result` into the `done` callback. The type/structure of the `result` must be agreed between the Integrator (or parent Feature App) and the Feature App.

Reply:
done

Reply:
Thanks. Can you please formate the comments to use 80 characters? I can recommend https://marketplace.visualstudio.com/items?itemName=stkb.rewrap for this task. And please also rebase the branch.